tools/bwa/bwa-mem.xml

```diff
@@ -212,21 +238,41 @@
   
     
     <conditional name="rg">
-      <param name="rg_selector" type="select" label="Set read groups information?" help="Specifying readgroup information can greatly simplify your downstream analyses by allowing combining multiple datasets. See help below for more details">
+      <param name="rg_selector" type="select" label="Set read groups information?" help="-R; Specifying readgroup information can greatly simplify your downstream analyses by allowing combining multiple datasets. See help below for more details">
         <option value="set">Set</option>
         <option value="do_not_set" selected="True">Do not set</option>
       </param>
       <when value="set">
-        <param name="ID" type="text" value="readgroup1" size="20" label="Specify readgroup ID" help="This value must be unique among multiple samples in your experiment">
+        <param name="ID" type="text" value="readgroup1" size="20" label="Read group identifier (ID)" help="This value must be unique among multiple samples in your experiment">
           <sanitizer invalid_char="">
             <valid initial="string.printable"/>
           </sanitizer>
         </param>
-        <param name="SM" type="text" value="blood" size="20" label="Specify readgroup sample name (SM)" help="This value should be descriptive">
+        <param name="SM" type="text" value="blood" size="20" label="Read group sample name (SM)" help="This value should be descriptive. Use pool name where a pool is being sequenced">
           <sanitizer invalid_char="">
             <valid initial="string.printable"/>
```

Comment:
string.letters + string.digits are probably enough. I would prefer to leave it like this but only take the first 20 (+x?) characters.